innovation_sweet_spots/pipeline/pilot/investment_predictions/create_dataset.py

```diff
@@ -8,17 +8,20 @@
     get_crunchbase_funding_rounds,
     get_crunchbase_ipos,
     get_crunchbase_acquisitions,
+    get_crunchbase_orgs,
 )
 import utils
 import pandas as pd
+from innovation_sweet_spots.analysis.wrangling_utils import CrunchbaseWrangler
+
 
 KEEP_COLS = [
     "id",
     "name",
     "legal_name",
-    "description",
+    "long_description",
```

Comment:
Just a heads up that it might be worthwhile including both `short_description` and `long_description` in the later analysis if you'll be using the description texts, because companies might not always have a `long_description`.

Reply:
...but that data can be always be fetched later, so probably don't need it here

Reply:
Thanks.
I was using `description` before when the company info was being loaded from `ISS_pilot_Crunchbase_companies.csv` but only `long_description` and `short_description` seem to available in `crunchbase_organisations.csv`. Did you create the `description` variable for `ISS_pilot_Crunchbase_companies.csv`?

Reply:
Yes indeed: I was (1) combining the `long_description` and `short_description` of companies into one string for the pilot analyses, and (2) analysing GtR and CB data simultaneously (eg, for topic modelling) so in the `description` column I also had `projectAbstract` data from GtR. Going forward, I'm not so sure about combining both datasets though.